core/src/test/java/info/rmapproject/core/rmapservice/impl/openrdf/OStatementsAdapterTest.java

```diff
@@ -0,0 +1,178 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This software was produced as part of the RMap Project (http://rmap-project.info),
+ * The RMap Project was funded by the Alfred P. Sloan Foundation and is a 
+ * collaboration between Data Conservancy, Portico, and IEEE.
+ *******************************************************************************/
+package info.rmapproject.core.rmapservice.impl.openrdf;
+
+import static java.net.URI.create;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.InputStream;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.Test;
+import org.openrdf.model.Resource;
+import org.openrdf.model.Statement;
+import org.openrdf.model.Value;
+import org.openrdf.model.ValueFactory;
+
+import info.rmapproject.core.CoreTestAbstract;
+import info.rmapproject.core.exception.RMapException;
+import info.rmapproject.core.model.impl.openrdf.ORAdapter;
+import info.rmapproject.core.model.impl.openrdf.ORMapDiSCO;
+import info.rmapproject.core.model.impl.openrdf.OStatementsAdapter;
+import info.rmapproject.core.rdfhandler.RDFType;
+import info.rmapproject.core.rdfhandler.impl.openrdf.RioRDFHandler;
+import info.rmapproject.core.vocabulary.impl.openrdf.RMAP;
+import info.rmapproject.testdata.service.TestDataHandler;
+import info.rmapproject.testdata.service.TestFile;
+
+public class OStatementsAdapterTest extends CoreTestAbstract {
+
+	//TODO: the asDisco tests are quick tests to catch exceptions, this does not do a full validation of results - need to add better validation
+	
+	private static final AtomicInteger counter = new AtomicInteger();
+		
+	/**
+	 * Test method for {@link info.rmapproject.core.model.impl.openrdf.OStatementsAdapter#asDisco(java.util.List, Supplier<URI>)}.
+	 * Runs a valid DiSCO through the asDisco process
+	 */
+	@Test
+	public void testAsDiscoValidDisco1() throws Exception {
+		InputStream stream = TestDataHandler.getTestData(TestFile.DISCOA_JSONLD);
+		RioRDFHandler handler = new RioRDFHandler();	
+		Set<Statement>stmts = handler.convertRDFToStmtList(stream, RDFType.get(TestFile.DISCOA_JSONLD.getType()), "");
+		ORMapDiSCO disco = OStatementsAdapter.asDisco(stmts,
+				() -> create("http://example.org/disco/" + counter.getAndIncrement()));
+		assertTrue(disco!=null);	

```

Comment:
assertNotNull may be better here (and in the other cases in this class); it's more explicit wrt what is being tested.